jake-solution.js

```diff
@@ -0,0 +1,8 @@
+var num = 0;
+
+for (i = 0; i < 1000; i++) {
```

Comment:
Should be `for (var i = 0; i < 1000; i++) {`; binds the scope of the variable `i` to the for loop.